src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -130,24 +130,24 @@ bool convert_precision(ov::pass::PassBase& pass,
     };
 
     auto convert_node_output_precision = [&](const std::shared_ptr<ngraph::Node>& node) {
+        bool res = false;
+        // Handle case with Constants as they can have consumers from other nGraph Function object
+        const auto constant = ov::as_type_ptr<opset10::Constant>(node);
+        const auto it = const_to_internal_output.find(node.get());
```

Comment:
nitpick: it's better to do this `find` as part of if condition because it will only perform in case of 
`constant && constant->get_output_element_type(0) == from` is true